lldb/test/Shell/SymbolFile/DWARF/x86/member-pointers.cpp

```diff
@@ -0,0 +1,48 @@
+// REQUIRES: lld
+
+// Microsoft ABI:
+// RUN: %clang_cl --target=x86_64-windows-msvc -c -gdwarf %s -o %t_win.obj
+// RUN: lld-link /out:%t_win.exe %t_win.obj /nodefaultlib /entry:main /debug
+// RUN: %lldb -f %t_win.exe -b -o "target variable mp1 mp2 mp3 mp4 mp5 mp6 mp7 mp8 mp9"
+//
+// DWARF has no representation of MSInheritanceAttr, so we cannot determine the size
+// of member-pointers yet. For the moment, make sure we don't crash on such variables.
+
+// Itanium ABI:
+// RUN: %clang --target=x86_64-pc-linux -gdwarf -c -o %t_linux.o %s
+// RUN: ld.lld %t_linux.o -o %t_linux
+// RUN: %lldb -f %t_linux -b -o "target variable mp1 mp2 mp3 mp4 mp5 mp6 mp7 mp8 mp9" | FileCheck %s

```

Comment:
@Michael137 You think that's ok as test-coverage also for non-MSVC?

Reply:
Yup, thanks!

Don't think we even need to link (in either case).

Also lets add a `CHECK-MSVC:` as well. That'd also notify us once that byte-size issue gets fixed on Windows

Reply:
Agree, added `CHECK-MSVC: error: Unable to determine byte size.` and dropped link-step for Itanium case. The latter doesn't seem to work for the MS case. It fails early with:
```
error: unable to resolve the module for file address 0x0 for variable 'mp1' in
S:\path\to\build\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\member-pointers.cpp.tmp_win.obj
```

Reply:
Ah fair enough, latest test changes LGTM